rudder-agent/SOURCES/Makefile

```diff
@@ -649,6 +649,8 @@ install-cfengine: build-cfengine $(INSTALL_DEPS) rudder.8.gz
 		LD_LIBRARY_PATH="$(DESTDIR)/opt/rudder/lib" ./$${binary} -M | gzip > $(DESTDIR)/opt/rudder/share/man/man8/$${binary}.8.gz; \
 	done
 	cd $(DESTDIR)/opt/rudder && rm -rf share/doc/cfengine
+	# Remove CFEngine systemd units (ours are in /lib/systemd)
+	cd $(DESTDIR) && rm -rf usr/lib/systemd/system/
```

Comment:
what if other systemd units have be installed in DESTDIR before ?
can't you remove files explicitly ?

Reply:
We don't use the same dir for systemd units, but we can name them indeed

Reply:
but other products in DESTDIR may do